tests/BatchedProcessorTest.php

```diff
@@ -196,21 +200,24 @@ public function testMultipleCommits()
         );
     }
 
-
     /**
      * Tests that the batch_soft_cap setting is properly respected
      */
     public function testSoftCap()
     {
+        $this->markTestSkipped(
```

Comment:
With the accompanied comment I feel like `markTestIncomplete` would probably be more ideal.